frontend/tests/integration/components/transportation/study-area-map-test.js

```diff
@@ -52,10 +52,12 @@ module('Integration | Component | transportation/study-area-map', function(hooks
   });
 
   test('it hovers, displays information', async function(assert) {
-    this.server.create('modal-split', { id: 'test' });
+    const geoid = '1';
+    this.server.create('census-tract', { geoid });
+    this.server.create('census-tract', { geoid });
```

Comment:
fwiw you could say this instead: `this.server.createList('census-tract', 2, { geoid });`

Reply:
<3 googled for this for hrs lol